plugins/modules/ec2_instance.py

```diff
@@ -922,6 +922,12 @@ def build_volume_spec(params):
             for int_value in ['volume_size', 'iops']:
                 if int_value in volume['ebs']:
                     volume['ebs'][int_value] = int(volume['ebs'][int_value])
+            if 'volume_type' in volume['ebs'] and volume['ebs']['volume_type'] == 'gp3':
+                if not volume['ebs'].get('iops'):
+                    volume['ebs']['iops'] = 3000
+                if 'throughput' in volume['ebs']:
```

Comment:
We need to call out that setting the throughput requires botocore >= 1.19.27

Reply:
@tremble Thank you for reviewing. Let me know please if this is what you meant.